app/templates/dayview.html

```diff
@@ -18,6 +18,11 @@
                 <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}" title="Zodiac-sign: {{zodiac.name}}" target="_blank"><img src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}" alt="zodiac sign" width="14em" height="13em"></a>
             </div>
             {% endif %}
+            {% if international_day %}
+            <div class="international-days">
```

Comment:
שים לב שהאופן שבו הכנסת את הסימן הוא על ידי שורה כתובה שפשוט תישאר שם לא משנה  לאן מדפדפים בתצוגת יום.
לצורך העניין הסמל של זודיאק הוא עיגול שמרחף מעל היום שאפשר ללחוץ עליו מתי שרוצים, אבל הוא לא מפריע להסתכל על אירועים
אני מציע לסדר את זה בצורה דומה - להוסיף לזה עיגול - ואולי במקום קישור לאתר חיצוני כמו הזודיאק, כשמצביעים אם העכבר על זה זה מתרחב ומציג את השורה.
אפשר לעשות את זה בCSS  בעזרת hover, transision וvisblity.

Reply:
It's okay because it's information about the day, and I don't think it disturbs